src/pybind/mgr/dashboard/module.py

```diff
@@ -405,6 +407,26 @@ def set_mgr_created_self_signed_cert(self):
         if result.retval != 0:
             return result
         return 0, 'Self-signed certificate created', ''
+    
+    @CLICommand("dashboard get issue")
+    def get_issues_cli(self, issue_number: str):
+        cephTrackerClient = CephTrackerClient()
+        response = cephTrackerClient.get_issues(issue_number)
+        return 0,str(response["issue"]), ''
+    
+    @CLICommand("dashboard create issue")
+    def report_issues_cli(self, project_id: str, tracker_id: str, subject: str, description: str, severity: Optional[str] = 'minor'):
+        '''
+        Create an issue in the Ceph Issue tracker
+        Syntax: ceph dashboard feedback create <project> <bug|feature> <subject> <description>
```

Comment:
The Syntax hint does not match the command invocation:
```
[root@ceph2 ceph]# ceph dashboard create issue
Invalid command: missing required parameter project_id(<string>)
dashboard create issue <project_id> <tracker_id> <subject> [<description>] [<severity>] :  Create an issue in the Ceph Issue tracker Syntax: ceph dashboard feedback create <project> <bug|feature> <subject> <description>
```
```
[root@ceph2 ceph]# ceph dashboard feedback create
no valid command found; 10 closest matches:
```

Reply:
Additionally, `tracker_id` is a bit misleading since I would expect it to be an `int` instead of either `bug` or `feature`. Maybe something like `tracker_type` or `tracker` alone as the tracker page uses.

Reply:
@pereman2 I think that `tracker_type_id` is a good choice.
What I found is you have to provide an integer on API but a string on CLI.
IMHO we should align both: either a number or a string.

Reply:
Changed it to `tracker`
We now have to enter project name string and tracker type string to create an issue using API